healthcare/api-client/v1/fhir/fhir_stores.py

```diff
@@ -0,0 +1,463 @@
+# Copyright 2018 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import os
+
+from googleapiclient import discovery
+
+
+# [START healthcare_get_client]
+def get_client():
+    """Returns an authorized API client.
+
+    Discovers the Healthcare API and
+    creates a service object using the service account credentials in the
+    GOOGLE_APPLICATION_CREDENTIALS environment variable.
+    """
+    api_version = "v1"
+    service_name = "healthcare"
+
+    return discovery.build(service_name, api_version)
+# [END healthcare_get_client]
+
+
+# [START healthcare_create_fhir_store]
+def create_fhir_store(project_id, cloud_region, dataset_id, fhir_store_id):
+    """Creates a new FHIR store within the parent dataset."""
+    client = get_client()
+    fhir_store_parent = "projects/{}/locations/{}/datasets/{}".format(
+        project_id, cloud_region, dataset_id
+    )
+
+    body = {"version": "STU3"}
+
+    request = (
+        client.projects()
+        .locations()
+        .datasets()
+        .fhirStores()
+        .create(parent=fhir_store_parent, body=body, fhirStoreId=fhir_store_id)
+    )
+
+    response = request.execute()
+    print("Created FHIR store: {}".format(fhir_store_id))
+
+    return response
+
+
+# [END healthcare_create_fhir_store]
+
+
+# [START healthcare_delete_fhir_store]
+def delete_fhir_store(project_id, cloud_region, dataset_id, fhir_store_id):
+    """Deletes the specified FHIR store."""
+    client = get_client()
+    fhir_store_parent = "projects/{}/locations/{}/datasets/{}".format(
+        project_id, cloud_region, dataset_id
+    )
+    fhir_store_name = "{}/fhirStores/{}".format(fhir_store_parent, fhir_store_id)
+
+    request = (
+        client.projects()
+        .locations()
+        .datasets()
+        .fhirStores()
+        .delete(name=fhir_store_name)
+    )
```

Comment:
I don't think we need the `()` (if black is forcing it you can ignore this)
```suggestion
    request = client.projects()
        .locations()
        .datasets()
        .fhirStores()
        .delete(name=fhir_store_name)
```

Reply:
Yes, black is requiring it.